hazelcast/src/main/java/com/hazelcast/cache/impl/operation/CacheExpireBatchBackupOperation.java

```diff
@@ -67,12 +67,23 @@ private void equalizeEntryCountWithPrimary() {
         if (diff > 0) {
             recordStore.sampleAndHardRemoveEntries(diff);
 
-            assert recordStore.size() == primaryEntryCount : String.format("Failed to remove %d entries while attempting "
-                    + "to match primary entry count %d, recordStore size is now %d",
+            assert recordStore.size() == primaryEntryCount : String.format("Failed"
+                            + " to remove % d entries while attempting to match"
```

Comment:
the extra space in "% d" will break formatting? (same space exists also in other occurrence of this assertion)

Reply:
good catch. fixed.